src/components/DateTimeField/index.jsx

```diff
@@ -74,10 +73,11 @@ class DateTimeField extends React.Component {
               />
             }
             placeholder="mm/dd/yyyy"
+            pattern="(0[1-9]|1[012])[- /.](0[1-9]|[12][0-9]|3[01])[- /.](19|20)\d\d"
```

Comment:
Has this been tested with international dates where they don't follow the order of MM/DD/YYYY? Like in the past with our international partners, the date field would automatically switch to being in DD/MM/YYYY and they could enter it in that way.

Reply:
@Dillon-Dumesnil I am going to guess this was done by the date-picker component which we were using from paragon.  the only thing we are really relying on here is the place holder to let the user know what the expected input is.  I could add something tho to detect where the user is geographically and then change the logic according to that.  what do you think?

Reply:
Yeah, I believe it was done by default with the date type we were using. I guess I'm hesitant to now have to change the behavior for all of our overseas partners. I think adding in logic about detecting the user geographically would be overkill for this. Just worth noting that we are now hardcoding in the American way of doing dates. Maybe at minimum, add in a comment to revisit this in the future and see if we can go back to using date.

Reply:
we are switching back to paragon which will support safari, supposedly soon